package.json

```diff
@@ -78,6 +78,10 @@
                 "title": "Take Screenshot"
             },
             {
+                "command": "ev3devBrowser.deviceTreeItem.printSysinfo",
```

Comment:
Might make more sense to call this `showSysinfo`. "show" is used many other places to mean "show something to the user".